estate/models/estate_property_offer.py

```diff
@@ -46,6 +46,21 @@ def _inverse_date_deadline(self):
                 offer.date_deadline - (offer.create_date.date() or fields.Date.today())
             ).days
 
+    @api.model
+    def create(self, offers_list):
```

Comment:
```suggestion
    def create(self, vals_list):
```
not very important but just FYI we usually name this as `vals_list`